### PR TITLE
feat: [AXIMST-172] Certificates page - certificates delete

### PR DESCRIPTION
**Pull request by khudym**
YT: https://youtrack.raccoongang.com/issue/AXIMST-172

### CODE REVIEW

File: src/certificates/certificates-card/certificate-signatories/signatory/Signatory.test.jsx [hunk 1]
```diff
@@ -71,4 +71,15 @@ describe('Signatory Component', () => {
...
+
+  it('shows confirm modal on delete icon click', async () => {
+    const { getByLabelText, getByText } = renderSignatory(
+      { ...defaultProps, componentMode: MODE_STATES.create, showDeleteButton: true },
+    );
+    const deleteIcon = getByLabelText(messages.deleteTooltip.defaultMessage);
+
+    userEvent.click(deleteIcon);
+
+    expect(getByText(`Delete "${defaultProps.name}" from the list of signatories?`)).toBeInTheDocument();
```
Comment (PKulkoRaccoonGang) [inline src/certificates/certificates-card/certificate-signatories/signatory/Signatory.test.jsx:83]: [important]: Please use an [existing](https://github.com/raccoongang/frontend-app-course-authoring/blob/6ef7e35e260f2274f878b1793aab1a78832ebdb2/src/certificates/certificates-card/messages.js#L126) translation. For [example](https://github.com/raccoongang/frontend-app-course-authoring/blob/6ef7e35e260f2274f878b1793aab1a78832ebdb2/src/course-team/info-modal/InfoModal.test.jsx#L63)

File: src/certificates/confirm-modal/ConfirmModal.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import { ActionRow, AlertModal, Button } from '@edx/paragon';
+
+const ConfirmModal = ({
```
Comment (PKulkoRaccoonGang) [inline src/certificates/confirm-modal/ConfirmModal.jsx:4]: [qiestion]: Can we use an already [existing generic component](https://github.com/raccoongang/frontend-app-course-authoring/blob/6ef7e35e260f2274f878b1793aab1a78832ebdb2/src/generic/modal-notification/index.jsx)?

**Review by PKulkoRaccoonGang — CHANGES_REQUESTED** [review]
@khudym Some observations on the alert reporting the absence of the Internet. It seems to me that we need to move this alert down from the modal window (since this is implemented in Legacy).

telegram-cloud-photo-size-2-5332538752725275834-y


telegram-cloud-photo-size-2-5332538752725275833-y

**Review by monteri — COMMENTED** [review]
LGTM, do fixes, that @PKulkoRaccoonGang proposed in the comments.

For future tasks, consider searching for generics first before implementing own soltion like.

File: src/certificates/certificates-card/certificate-signatories/signatory/Signatory.test.jsx (hunk 1 above)
Comment (PKulkoRaccoonGang) [reply src/certificates/certificates-card/certificate-signatories/signatory/Signatory.test.jsx:111]: You have already added a check for this text below. Do we need the legacy version?

File: src/certificates/scss/Certificates.scss [hunk 3]
```diff
@@ -79,3 +79,7 @@
     }
   }
 }
+
+.certificates.alert-toast {
+  z-index: 9999;
```
Comment (PKulkoRaccoonGang) [inline src/certificates/scss/Certificates.scss:84]: `  z-index: $zindex-popover;`

**Review by PKulkoRaccoonGang — APPROVED** [review]

**Review by GlugovGrGlib — APPROVED** [review]

**Pull request merged**